src/registrar/templates/application_status.html

```diff
@@ -22,7 +22,7 @@ <h1>Domain request for {{ domainapplication.requested_domain.name }}</h1>
           {% if domainapplication.status == 'approved' %} Approved 
           {% elif domainapplication.status == 'in review' %} In Review 
           {% elif domainapplication.status == 'rejected' %} Rejected
-          {% elif domainapplication.status == 'submitted' %} Received
+          {% elif domainapplication.status == 'submitted' %} Submitted
```

Comment:
@zandercymatics This is a great suggestion!

Reply:
Yeah I agree we could totally do that! I had actually wondered if this was just so that we could have an easy way to have a different name for each status and make it more human readable. That does sound like a nice add on later